scripts/seed.js

```diff
@@ -0,0 +1,72 @@
+const fetch = require("node-fetch");
+const fs = require("fs");
+const os = require("os");
+const path = require("path");
+const { crc8 } = require("node-crc");
+const { Crypto } = require("node-webcrypto-ossl");
+
+global.crypto = new Crypto();
+
+const config = require("../dfx.json");
+const userLib = require(path.join(
+  os.homedir(),
+  "/.cache/dfinity/versions/",
+  config.dfx,
+  "/js-user-library/src"
+));
+const {
+  generateKeyPair,
+  HttpAgent,
+  makeActorFactory,
+  makeAuthTransform,
+  makeNonceTransform,
+} = userLib;
+
+// Main
+
+const main = async () => {
+  console.log("Adding profiles...");
+  const profiles = require("./data");
+  profiles.forEach(async (profile) => {
+    const linkedup = await getActor("linkedup");
+    const userId = await linkedup.create(profile);
+    console.log("...profile added", userId);
+  });
```

Comment:
We want a different actor for each profile because we want a different identity for each user and that's pulled from the actor.

Reply:
@stanleygjones Does it help if `linkedup.create` can take a principal id as string directly? `getActor` is clever but also feels hacky and not robust.